ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -36,12 +39,15 @@ internal constructor(
   private val surveyRepository: SurveyRepository
 ) : AbstractViewModel() {
 
-  val openDrawerRequests: @Hot FlowableProcessor<Nil> = PublishProcessor.create()
+  private val _openDrawerRequests: MutableSharedFlow<Nil> = MutableSharedFlow()
+  val openDrawerRequestsFlow: SharedFlow<Nil> =
+    _openDrawerRequests.shareIn(viewModelScope, SharingStarted.Lazily, replay = 0)
```

Comment:
I think here you can just use `asSharedFlow` instead. I believe emissions will automatically occur in the ViewModelScope because of the `launch` call below.

Reply:
Thanks! Done